attributes/objects/pdf.json

```
@@ -1 +1,11 @@
-{}
\ No newline at end of file
+{
+    "filename": "#pdf-filename",
+    "url": "#pdf-url",
+    "page": {
+        "value-pattern": "\\d"
```

'\d'은 digit 한개만 match 됩니다. 즉, 10은 안되는거죠.
```suggestion
        "value-pattern": "[1-9][0-9]*"
```